android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -43,6 +43,11 @@ open class MainActivity : ReactActivity() {
     }
 
     private fun handleSend(intent: Intent) {
+        // Intent is reused after quitting, skip it.
+        if((getIntent().flags and Intent.FLAG_ACTIVITY_LAUNCHED_FROM_HISTORY)!=0){
```

Comment:
nit: spaces -- see the style of the existing code

(In Android Studio you should be able to hit Ctrl+Alt+L and have it reformat the code for you. In steady state that should have no effect.)